apps/omg_watcher_rpc/lib/web/validators/transaction_constraints.ex

```diff
@@ -25,19 +25,29 @@ defmodule OMG.WatcherRPC.Web.Validator.TransactionConstraints do
   @spec parse(%{binary() => any()}) :: {:ok, Keyword.t()} | {:error, any()}
   def parse(params) do
     constraints = [
-      {"address", [:address, :optional]},
-      {"blknum", [:pos_integer, :optional]},
-      {"metadata", [:hash, :optional]},
-      {"limit", [:pos_integer, :optional]},
-      {"page", [:pos_integer, :optional]}
+      {"address", [:address, :optional], :address},
+      {"blknum", [:pos_integer, :optional], :blknum},
+      {"metadata", [:hash, :optional], :metadata},
+      {"txtypes", [list: &to_tx_type/1, optional: true], :txtypes},
+      {"limit", [:pos_integer, :optional], :limit},
+      {"page", [:pos_integer, :optional], :page}
     ]
 
-    Enum.reduce_while(constraints, {:ok, []}, fn {key, validators}, {:ok, list} ->
+    Enum.reduce_while(constraints, {:ok, []}, fn {key, validators, atom}, {:ok, list} ->
```

Comment:
is this closing that issue (atoms didn't exist)?

Reply:
Not the previous issue that @unnawut mentioned a few days ago, that is already closed: https://github.com/omisego/elixir-omg/issues/1301.
However, I added atoms here to prevent this issue to appear again.